packages/amplify-category-analytics/src/provider-utils/awscloudformation/cloudformation-templates/pinpoint-cloudformation-template.json

```diff
@@ -134,7 +125,10 @@
               "Statement": [
                 {
                   "Effect": "Allow",
-                  "Action": ["mobileanalytics:*", "mobiletargeting:*"],
+                  "Action": [
+                    "mobileanalytics:*",
+                    "mobiletargeting:*"
+                  ],
                   "Resource": "*"
```

Comment:
this can also be scoped down

Reply:
Agreed but keeping this PR scoped to just cloudwatch